rmw_fastrtps_shared_cpp/include/rmw_fastrtps_shared_cpp/custom_participant_info.hpp

```diff
@@ -130,77 +138,107 @@ typedef struct CustomParticipantInfo
     EventListenerInterface * event_listener);
 } CustomParticipantInfo;
 
-class ParticipantListener : public eprosima::fastdds::dds::DomainParticipantListener
-{
+class ParticipantListener
+    : public eprosima::fastdds::dds::DomainParticipantListener {
 public:
-  explicit ParticipantListener(
-    const char * identifier,
-    rmw_dds_common::Context * context)
-  : context(context),
-    identifier_(identifier)
-  {}
+  explicit ParticipantListener(const char *identifier,
+                               rmw_dds_common::Context *context,
+                               const std::string &hostname,
+                               const rmw_discovery_options_t *discovery_options)
+      : context(context), identifier_(identifier), my_hostname_(hostname),
+        discovery_options_(*discovery_options) {}
```

Comment:
nitpick: it's weird that we changed the style as we did this. It makes it hard to understand what actually changed due to the whitespace change.

Reply:
Removed whitespace changes in `custom_participant_info.hpp` in 65de1fb